docs/standard/microservices-architecture/architect-microservice-container-applications/direct-client-to-microservice-communication-versus-the-API-Gateway-pattern.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
file name here still has uppercase. also these ebook articles have really long names close to the maximum allowed.

This url won't even fit in the address bar:
http://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/architect-microservice-container-applications/direct-client-to-microservice-communication-versus-the-API-Gateway-pattern

we should really come up with a better way of naming these articles. /cc @BillWagner

Reply:
👍  That was because when pasting in Windows it doesn't change it if it is not case-sensitive. Fixed now.